docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -417,47 +418,59 @@ You can also test a version of your collection in development by installing it f
 
 .. _publishing_collections:
 
-Publishing collections
-----------------------
+Distributing collections
+========================
+
+You can distribute your collections by publishing them on a distribution server. Distribution servers include Ansible Galaxy, Red Hat Automation Hub, and privately hosted Automation Hub instances. You can publish any collection to Ansible Galaxy and/or to a privately hosted Automation Hub instance. If your collection is certified by Red Hat, you can publish it to the Red Hat Automation Hub.
+
+Prerequisites
+-------------
+
+1. Get a namespace on each distribution server you want to use (Galaxy, private Automation Hub, Red Hat Automation Hub)
+2. Get an API token for each distribution server you want to use
+3. Specify your API token(s)
 
-You can publish collections to Galaxy using the ``ansible-galaxy collection publish`` command or the Galaxy UI itself. You need a namespace on Galaxy to upload your collection. See `Galaxy namespaces <https://galaxy.ansible.com/docs/contributing/namespaces.html#galaxy-namespaces>`_ on the Galaxy docsite for details.
+Getting a namespace
+^^^^^^^^^^^^^^^^^^^
 
-.. note:: Once you upload a version of a collection, you cannot delete or modify that version. Ensure that everything looks okay before you upload it.
+You need a namespace on Galaxy and/or Automation Hub to upload your collection. To get a namespace:
+
+* For Galaxy, see `Galaxy namespaces <https://galaxy.ansible.com/docs/contributing/namespaces.html#galaxy-namespaces>`_ on the Galaxy docsite for details.
+* For Automation Hub, see the `Ansible Certified Content FAQ <https://access.redhat.com/articles/4916901>`_.
 
 .. _galaxy_get_token:
 
 Getting your API token
 ^^^^^^^^^^^^^^^^^^^^^^
 
-To upload your collection to Galaxy, you must first obtain an API token (``--token`` in the ``ansible-galaxy`` CLI command or ``token`` in the :file:`ansible.cfg` file under the ``galaxy_server`` section). The API token is a secret token used to protect your content.
+You need an API token for Galaxy and/or Automation Hub to upload your collection. Use the API token(s) to authenticate your connection to the distribution server(s) and protect your content.
 
 To get your API token:
 
 * For Galaxy, go to the `Galaxy profile preferences <https://galaxy.ansible.com/me/preferences>`_ page and click :guilabel:`API Key`.
-* For Automation Hub, go to https://cloud.redhat.com/ansible/automation-hub/token/ and click :guilabel:`Load token` from the version dropdown.
+* For Automation Hub, go to `the token page <https://cloud.redhat.com/ansible/automation-hub/token/>`_ and click :guilabel:`Load token` from the version dropdown.
```

Comment:
```suggestion
* For Automation Hub, go to `the token page <https://cloud.redhat.com/ansible/automation-hub/token/>`_ and click :guilabel:`Load token`.
```

Reply:
Just looked and there's no version dropdown, just a button labeled load token.

Reply:
Better and better!